ACMAS/app/ACMAS_Web/models.py

```diff
@@ -2,23 +2,34 @@
 
 
 class Question(models.Model):
+    # File_Name = models.TextField()
+    # Question = models.TextField()
```

Comment:
Why are all these in comments? Probably best to remove if not needed. Or remove in next commit.

Reply:
Yeah, I added these when I was trying to fix the names of columns in the db. I didn't want to delete the work and ended up forgetting they were here. I'll move these comments to a new branch where I'll deal with the changes I was aiming to complete.

Reply:
Resolved